<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trivial Calc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="edParam1"/>
    </item>
    <item>
     <widget class="QComboBox" name="cbOperation">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="maxVisibleItems">
       <number>4</number>
      </property>
      <property name="maxCount">
       <number>4</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::NoInsert</enum>
      </property>
      <property name="minimumContentsLength">
       <number>4</number>
      </property>
      <item>
       <property name="text">
        <string>+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>*</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>/</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edParam2"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnCalc">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edResult">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCalc</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculate()</slot>
 </slots>
</ui>
